boards/arm/disco_l475_iot1/disco_l475_iot1_defconfig

```diff
@@ -41,17 +37,14 @@ CONFIG_UART_CONSOLE=y
 
 #enable pwm
 CONFIG_PWM=y
-CONFIG_PWM_STM32=y
 
 #enable MPU
 CONFIG_STM32_ARM_MPU_ENABLE=y
 
 #enable I2C
 CONFIG_I2C=y
-CONFIG_I2C_STM32_V2=y
 
 CONFIG_SPI=y
-CONFIG_SPI_STM32=y
 CONFIG_SPI_STM32_INTERRUPT=y
 CONFIG_SPI_3=y
 CONFIG_SPI_3_IRQ_PRI=10
```

Comment:
Thanks for the update! I'm confused though: IRQ_PRI is still in this board file, but you said only Carbon should be left. Did you get them all?

Reply:
I am sorry, I mentioned the one I added. This was already there. I remove them both in the PR which enables STM32 SPI devices to get their configuration from DT.

Reply:
Ah, okay, thanks! That makes sense.